cukinia

```diff
@@ -203,6 +203,13 @@ logging()
 	return 0
 }
 
+getch()
```

Comment:
* add a small docstring
* prefix the function with _ (eg. like `_time_elapsed`) as it's mostly an internal and is not designed to be exposed
* move before `_time_elapsed()`